src/tactic/mk_opaque.lean

```diff
@@ -0,0 +1,49 @@
+/-
```

Comment:
This file is no longer needed, right? This is just a drop in replacement for `clear_value`? (I prefer the name `clear_value` to `mk_opaque`.)

Reply:
That's right, I'll delete the file

Reply:
I assume your request for testing dependencies no longer applies. I think it is already tested appropriately in `clear_value`